.circleci/config.yml

```diff
@@ -99,9 +99,9 @@ jobs:
       - build_app:
           template: 'Autocomplete.js 0'
           app: 'autocomplete.js'
-      - build_app:
-          template: 'InstantSearch.js'
-          app: 'instantsearch.js'
+      # - build_app:
+      #     template: 'InstantSearch.js'
+      #     app: 'instantsearch.js'
```

Comment:
This doesn't solve the issue with this CI job. I have the same for Vue InstantSearch as well on my PR.

I wonder if we should drop it altogether. node 10 is EOL since mid 2021.

Alternatively, we could update the tests to run on the 2 current LTS versions (16 and 14), to make it more relevant to real use. 

I can work on that in a separate PR. Wdyt?

Reply:
sounds good, but our next release will need to a major then (no big deal, just put BREAKING CHANGE in the commit)